packages/grid/x-data-grid-pro/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mui/x-data-grid-pro",
   "version": "5.7.0",
-  "description": "The commercial edition of the data grid component (MUI X).",
+  "description": "The Pro plan edition of the data grid component (MUI X).",
   "author": "MUI Team",
   "main": "src/index.ts",
   "license": "SEE LICENSE IN LICENSE.txt",
```

Comment:
**Off-topic** Well, this one is funny.

1. first the file extension is wrong, it's without `.txt`

```suggestion
  "license": "SEE LICENSE IN LICENSE",
```

Proof: https://github.com/mui/mui-x/blob/3d2ff099f4407a3cf88b49f63910c80a88e5882e/packages/grid/x-data-grid-pro/LICENSE

Oops my bad.

2. Second, the license is gone. Compare https://unpkg.com/browse/@mui/x-data-grid-pro@5.5.0/ (OK) and https://unpkg.com/browse/@mui/x-data-grid-pro@5.6.0/ (KO) cc @flaviendelangle